src/corelibs/U2View/src/ov_sequence/ADVSingleSequenceWidget.cpp

```diff
@@ -628,6 +628,9 @@ void ADVSingleSequenceWidget::sl_onSelectRange() {
             getSequenceContext()->getSequenceSelection()->setSelectedRegions(curRegions);
         }
     }
+    CHECK(fw != nullptr, );
+
+    fw->setFocus();
```

Comment:
The originally focused widget may be already destroyed: you should use QPointer.

Reply:
A better solution would be not changing the code here at all: the focus must be returned to the original widget automatically and if it is not returned: this is a bug.

In your test you can close "Pan View" first, so the focus will remain in DetView.

Reply:
I checked on Linux and if I 

a) close PanView 
and 
b) click on DetView 

between steps 2 and 3 the focus after step 6 will remain on the DetView

Reply:
Well, OK, but the problem is still with us, what shell we do? Just skip it?

Reply:
Lets create a bug with this issue description and fix focus management in sequence view. It may be a small or a big fix. This requires investigation and a solution that may be very different from the current one.

Reply:
https://ugene.dev/tracker/browse/UGENE-7673